### PR TITLE
Fix intermediate files published

### LINKED ISSUE #680: Freebays intermediate files are published
### Description of the bug

cf https://nf-co.re/sarek/results#sarek/results-5bb160ddb2cee50c811585e450b16cba13c95a02/germline_test/freebayes/

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by maxulysse**
Will close #680

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by maxulysse** [comment]
Good news, as I extended the checks, I could see that only Freebayes was producting unexpected files

**Review by mahesh-panchal — APPROVED** [review]

File: conf/modules.config [hunk 1]
```diff
@@ -607,6 +607,9 @@ process{
         ext.prefix       = { meta.num_intervals <= 1 ? "${meta.id}" : "${meta.id}.${target_bed.simpleName}" }
         ext.args         = '--min-alternate-fraction 0.1 --min-mapping-quality 1'
         ext.when         = { params.tools && params.tools.split(',').contains('freebayes') }
+        publishDir       = [
```
Comment (FriederikeHanssen) [inline conf/modules.config:610]: why is this happening though? shouldn't publishing be disabled by default?

Comment (maxulysse) [reply conf/modules.config:610]: I agree but it was

Comment (FriederikeHanssen) [reply conf/modules.config:610]: I believe you :D just wondering if something else is going on somewhere.

Comment (maxulysse) [reply conf/modules.config:610]: I'm sure something else is going on. I'll check it more more in detail next week

**Pull request merged**